src/manager/MapFishPrintV3Manager.js

```diff
@@ -387,7 +387,8 @@ export class MapFishPrintV3Manager extends BaseMapFishPrintManager {
           layers: serializedLayers,
           projection: mapProjection.getCode(),
           rotation: this.calculateRotation() || 0,
-          scale: this.getScale()
+          scale: this.getScale(),
+          longitudeFirst: mapProjection.getUnits() !== 'm',
```

Comment:
Not sure if this assumption is always correct as the axis order is also defined by the service version (see [here](https://docs.geoserver.org/master/en/user/services/wms/basics.html#axis-ordering)). I would suggest to fix the present TODO to pass any required parameter as needed.

Reply:
Ah, ok, didn't recognize it, you're right of course. Will close this now and address your notes in a separate PR to keep history clearer. Thanks for the hint!

Reply:
s. here https://github.com/terrestris/mapfish-print-manager/pull/190